### ISSUE TITLE
Show code coverage in report

### ISSUE THREAD

**Issue by ISErikMeijer** [stryker-mutator/stryker-net#2662]
If possible I would like if it showed the normal overall code coverage in the final stryker report, so I don't have to separately run a code coverage test.

**Comment by richardwerkman** [comment]
We might be able to reuse the coverlet collector https://github.com/coverlet-coverage/coverlet/blob/master/src/coverlet.collector/DataCollection/CoverletCoverageCollector.cs

**Comment by dupdob** [comment]
We could capture code coverage as part of the initial test run (that could be used as a normal test run BTW), but no use benefit beyond that. I am probably missing something important, but:
1. As pointed above, code coverage results for mutated code may not reflect coverage for normal code
2. Coverlet does not capture coverage per test, only per session. This would significantly impair coverage based optimization, as we could only determine which mutants are tested, without knowing which test(s) cover(s) them

There are also significant hurdle to address
1. Ensure assembly PDBs contain location information for mutation (as of today, mutated code bears no location information)
2. Parse coverage result to determine mutant coverage

**Comment by richardwerkman** [comment]
@dupdob I don't think we need to use the coverage information to determine mutant coverage. We can keep that like it is. We just need to add coverage analysis to the initial testrun and get the percentage from it. The initial testrun doesn't contain mutations so the coverage should be correct.
